README.md

```diff
@@ -0,0 +1,61 @@
+# Money Transfer Project
+
+## Prerequisites
+
+Before running this application, ensure you have the following installed:
+
+* [.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later
+* Temporal Server
+
+## Steps to get started
+
+## Start the Temporal Server
+Run the command: 
+`temporal server start-dev`
+
+## Clone the Repo
+Follow these steps to copy to you local machine. 
+
+```
+git clone https://github.com/jsundai/money-transfer-temporal-template-dotnet.git
+
+cd money-transfer-temporal-template-dotnet
+```
+
+## Restore Dependencies 
+Run the following command: 
+```
+dotnet restore 
+```
+
+## Run The Workflow 
+Run the following commands to initiate the Workflow that starts the money transfer process.
+
+1. **Navigate to the Client folder** 
+
+   ```
+   cd MoneyTransferClient
+   ```
+   
+1. **Run the following command to begin the client**
+
+   ```
+   dotnet run
+   ```
```

Comment:
May be able to just `dotnet run --project MoneyTransferClient` instead of requiring them to `cd` (same for worker)